MRFIXIT: Change the power menu to display "Exit to Batocera"
--- a/addons/skin.estuary/xml/DialogButtonMenu.xml	2018-04-20 21:39:33.920915856 -0400
+++ b/addons/skin.estuary/xml/DialogButtonMenu.xml	2018-04-20 21:42:32.904915856 -0400
@@ -18,7 +18,7 @@
 				<include>ButtonMenuList</include>
 				<content>
 					<item>
-						<label>$LOCALIZE[13012]</label>
+						<label>Exit to Batocera</label>
						<onclick>Quit()</onclick>
 						<visible>System.ShowExitButton</visible>
 					</item>

